modules/apps/portal-vulcan/portal-vulcan-api/src/main/java/com/liferay/portal/vulcan/util/UriInfoUtil.java

```diff
@@ -53,17 +73,19 @@ private static boolean _isHttpsEnabled() {
 	}
 
 	private static UriBuilder _updateUriBuilder(UriBuilder uriBuilder) {
-		if (!Validator.isBlank(PortalUtil.getPathContext())) {
-			URI uri = uriBuilder.build();
+		URI uri = Objects.requireNonNull(uriBuilder.build());
```

Comment:
Maybe it is because of my lack of context here but, why are you checking the URI here instead of inside the following if?
I am asking this because the URI is only used inside the if
`if (!Validator.isBlank(PortalUtil.getPathContext()))` 
Should be better to check for it inside and avoid possible NPE?

Reply:
Good point, fixing.